src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2083,19 +2083,18 @@ limitations under the License.
         </clusters>
     </deviceType>
     <deviceType>
-        <name>MA-boolean-sensor</name>
+        <name>MA-water-freeze-detector</name>
         <domain>CHIP</domain>
-        <typeName>Matter Boolean Sensor</typeName>
+        <typeName>Matter Water Freeze Detector</typeName>
         <profileId editable="false">0x0103</profileId>
         <deviceId editable="false">0x0041</deviceId>
         <class>Simple</class>
         <scope>Endpoint</scope>
         <clusters lockOthers="true">
             <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true"></include>
             <include cluster="Boolean State" client="false" server="true" clientLocked="true" serverLocked="true"></include>
-            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
-                <requireAttribute>TAG_LIST</requireAttribute>
-            </include>
+            <include cluster="Boolean State Configuration" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```

Comment:
I might be wrong here, does serverLocked indicate if this is mandated or not, in that case, this should be false?

Reply:
You are right. Will update it.

Reply:
I see all mixed use of these properties. I don't think everyone understands how to use these.